content/post/azure-monitor-terraform.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Azure Monitor & Terraform: Collecting custom log files"
+date: 2023-10-03T14:46:23+01:00
+hero: /images/hero-3.jpg
+excerpt: "Working with Azure Monitor in collecting custom log files using Terraform"
+timeToRead: 5
+authors:
+  - Dylan McGinley
+draft: true
+license: cc-by
+---
+To collect custom log files from a VM (Virtual Machine) managed by Azure, it is recommended to use Azure Monitor and the Azure Monitor Agent (AMA). The Azure Monitor Agent is the successor to the Log Analytics Agent which is due to be deprecated on August 31, 2024. There are currently issues associated with the azurerm provider in relation to Azure Monitor.
+
+I suspect in the future these issues will be addressed with updates but for now this post will serve to provide a workaround along with an explanation as to what these are.
+
+Azurerm version 3.74.0
```

Comment:
Is this a heading?

Reply:
I think this one is a header of a section, h1 or h2 is good, but it depends on how the webpage will show it.